skimage/transform/_hough_transform.pyx

```diff
@@ -97,8 +98,8 @@ def _hough_circle(cnp.ndarray img,
     return acc
 
 
-def _hough_ellipse(cnp.ndarray img, int threshold=4, double accuracy=1,
-                   int min_size=4, max_size=None):
+def _hough_ellipse(cnp.ndarray img, Py_ssize_t threshold=4, double accuracy=1,
+                   Py_ssize_t min_size=4, max_size=None):
```

Comment:
Given the use of `min_size` below, I think this should actually be a double? The use of `Py_ssize_t` for `threshold` is also dubious...? These do not refer to array indices or sizes...

Reply:
`min_size` is compared to a C `double` but defined/documented as an integer, which might be intentional. It is a size/length, so I used `Py_ssize_t` for consistency but I'm OK with reverting to int.

`threshold` is compared to the max of a histogram count so `Py_ssize_t` should be correct.

Reply:
Well, in my interpretation, `Py_ssize_t` is specifically for sizes of collections, arrays, etc., and corresponding indices, rather than any conceptual count. However I'm not super-fussed about this.

Reply:
The maximum count in a histogram is the size of the input array. It's not any conceptual count.